__tests__/test_command_stream.py

```diff
@@ -0,0 +1,84 @@
+import unittest
+from unittest.mock import patch
+import json
+
+from command_stream import handle_command
+from dialog.dialog import ProcessSMSMessage, TriggerReminder, StartDrill
+
+
+class TestHandleCommand(unittest.TestCase):
+    @patch("command_stream.process_command")
+    def test_inbound_sms(self, process_command_mock):
+        with open("sample_events/inbound_sms_event.json") as f:
+            mock_kinesis_event = json.load(f)
+
+        handle_command(mock_kinesis_event, None)
+
+        process_command_mock.assert_called_once()
+
+        call = process_command_mock.call_args_list[0]
+        args, _ = call
+
+        command = args[0]
+        self.assertTrue(isinstance(command, ProcessSMSMessage))
+        self.assertEqual(command.phone_number, "+14802865415")
+        self.assertEqual(command.content, "🤡")
+
+        self.assertEqual(
+            args[1], mock_kinesis_event["Records"][0]["kinesis"]["sequenceNumber"]
+        )
+
+    @patch("command_stream.process_command")
+    def test_trigger_reminder(self, process_command_mock):
+        with open("sample_events/trigger_reminder_command.json") as f:
+            mock_kinesis_event = json.load(f)
+
+        handle_command(mock_kinesis_event, None)
+
+        process_command_mock.assert_called_once()
+
+        call = process_command_mock.call_args_list[0]
+        args, _ = call
+
+        command = args[0]
+        self.assertTrue(isinstance(command, TriggerReminder))
+        self.assertEqual(command.prompt_slug, "hand-washing")
+        self.assertEqual(command.drill_id, "1234-1234-1234-1234")
```

Comment:
the `drill_id` field has gone away. I'd use another one. I'll probably get around to cleaning it up when I actually write those tests.